lib/utapi/utilities.js

```diff
@@ -126,16 +126,26 @@ export function listBucketMetrics() {
  * @param {object} log - werelogs logger
  * @param {object} metricObj - the object containing the relevant data for
  * pushing metrics in Utapi
- * @param {string} metricObj.bucket - bucket name
- * @param {string} metricObj.byteLength - [optional] current object size (used,
+ * @param {string} metricObj.bucket - (optional) bucket name
```

Comment:
you can put in brackets around `metricObj.bucket` to indicate it's optional
http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values

same for the other optional params